lexicons/app/bsky/feed/defs.json

```diff
@@ -243,13 +243,18 @@
             "app.bsky.feed.defs#interactionRepost",
             "app.bsky.feed.defs#interactionReply",
             "app.bsky.feed.defs#interactionQuote",
-            "app.bsky.feed.defs#interactionShare"
+            "app.bsky.feed.defs#interactionShare",
+            "app.bsky.feed.defs#interactionVideoWatch"
           ]
         },
         "feedContext": {
           "type": "string",
           "description": "Context on a feed item that was originally supplied by the feed generator on getFeedSkeleton.",
           "maxLength": 2000
+        },
+        "videoWatchMs": {
```

Comment:
Would it make sense to use a more generic name here ?

For example: `interactionDurationMs` could also be for `interactionSeen`. This metric could also be usefull if we add an interaction that describes how long the user spent reading the responses threads.